frontend/src/components/home/HomePage.tsx

```diff
@@ -136,7 +136,6 @@ export default class HomePage extends React.Component<HomePageProps, HomePageSta
             isLoading: false,
             displayUpdateNotice: false,
           });
-          this.checkUpdate();
```

Comment:
@Ishirak: Why is this line removed? If the checkUpdate is not called after fetching commits there could be short (<10s) inconsistency between the commits table and the CommitPanel.